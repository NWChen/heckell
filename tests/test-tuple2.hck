let f : int * int -> bool;
f(x, y) = x < y end

let g : int -> real * int;
g(x) = (x/1.234, 2*x) end

let h : int * real * bool -> real * bool * int;

let t : int * (real -> real) * int;

let v : (int -> int) * (bool -> real * real) *int;
