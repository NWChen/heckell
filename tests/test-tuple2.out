let f: ((int * int) -> bool)
f = (x,y) ->
  (
    x < y
  )
let g: (int -> (real * int))
g = (x) ->
  (
    (x / 1.234, 2 * x)
  )
let h: ((int * real * bool) -> (real * bool * int))
let t: (int * (real -> real) * int)
let v: ((int -> int) * (bool -> (real * real)) * int)
